doc/nrf/protocols/matter/getting_started/memory_profiling.rst

```diff
@@ -0,0 +1,371 @@
++---------------------+------------------------------------------------------------------+----------------------------------------------------------------+
+| wpa_supplicant_wq   | :kconfig:option:`CONFIG_WPA_SUPP_WQ_STACK_SIZE`                  | WPA supplicant work queue thread                               |
+|                     |                                                                  | Processing Wi-Fi task queue                                    |
+|                     |                                                                  | (Matter over Wi-Fi only)                                       |
++---------------------+------------------------------------------------------------------+----------------------------------------------------------------+
+| nrf700x_bh_wq       | :kconfig:option:`CONFIG_NRF700X_BH_WQ_STACK_SIZE`                | nRF700x Wi-Fi driver work queue                                |
++---------------------+------------------------------------------------------------------+----------------------------------------------------------------+
+| nrf700x_intr_wq     | :kconfig:option:`CONFIG_NRF700X_IRQ_WQ_STACK_SIZE`               | Interrupts processing generated by the nRF700X Wi-Fi radio     |
++---------------------+------------------------------------------------------------------+----------------------------------------------------------------+
+| mbox_wq             | :kconfig:option:`CONFIG_IPC_SERVICE_BACKEND_RPMSG_WQ_STACK_SIZE` | Inter Processor Communication                                  |
+|                     |                                                                  | (For multi-processors targets only)                            |
++---------------------+------------------------------------------------------------------+----------------------------------------------------------------+
+
+Heap size
+---------
+
+The Matter application defines the static size of the heap used by all memory allocations.
+This configuration ignores the :kconfig:option:`CONFIG_HEAP_MEM_POOL_SIZE` Kconfig option value in the application configuration.
+The static size is determined by the :kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP` and :kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP_OVERRIDE` Kconfig options.
+You can set them to ``n`` to start using a dynamic heap size on your Matter device.
+
+The default static heap size means that you can define the maximum heap size for your application by setting the :kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP_SIZE` Kconfig value.
+You can also adjust the heap dedicated for mbedTLS purposes by setting the :kconfig:option:`CONFIG_MBEDTLS_HEAP_SIZE` Kconfig option value.
+
+Memory profiling troubleshooting
+================================
+
+While manipulating memory sizes, you may experience various bugs and faults on your device.
```

Comment:
Bugs sound scary :smile: It's not a bug if someone uses too small stack, so maybe let's call it an issue?  
```suggestion
While manipulating memory sizes, you may experience various issues and faults on your device.
```

Reply:
ok 😄